pkg/component/component.go

```diff
@@ -21,20 +21,22 @@ import (
 // componentSourceURLAnnotation is an source url from which component was build
 // it can be also file://
 const componentSourceURLAnnotation = "app.kubernetes.io/url"
+const componentSourceTypeAnnotation = "app.kubernetes.io/component-source-type"
 
 // ComponentInfo holds all important information about one component
 type ComponentInfo struct {
 	Name string
 	Type string
 }
 
-func CreateFromGit(client *occlient.Client, name string, ctype string, url string, applicationName string) error {
+func CreateFromGit(client *occlient.Client, name string, ctype string, url string, applicationName string, sourceType string) error {
 	labels := componentlabels.GetLabels(name, applicationName, true)
 	// save component type as label
 	labels[componentlabels.ComponentTypeLabel] = ctype
 
 	// save source path as annotation
 	annotations := map[string]string{componentSourceURLAnnotation: url}
+	annotations[componentSourceTypeAnnotation] = sourceType
```

Comment:
I think that we can hardcode this as if the `CreateFromGit` is called `sourceType` will always be `git`, isn't it?